src/extension/_locales/en/messages.json

```diff
@@ -633,6 +633,27 @@
   "user_switch": {
     "message": "Switch user"
   },
+  "v7_context_menu": {
+    "message": "Install the new AEM Sidekick"
+  },
+  "v7_hint_cover": {
+    "message": "https://www.aem.live/tools/sidekick/en/v7-teaser.png"
+  },
+  "v7_hint_description": {
+    "message": "A brand-new version of the AEM Sidekick extension is now available in the Chrome Web Store. Switch today for enhanced performance, new features, and a seamless authoring experience."
+  },
+  "v7_hint_title": {
+    "message": "Introducing the new AEM Sidekick"
+  },
+  "v7_install_later": {
+    "message": "Later"
+  },
+  "v7_install_now": {
+    "message": "Install now"
+  },
+  "v7_install_url": {
+    "message": "https://chromewebstore.google.com/detail/aem-sidekick-nextgen/igkmdomcgoebiipaifhmpfjhbjccggml"
```

Comment:
@dylandepass Because the name is part of the URL, I think we need to remove the `nextgen` suffix from the name to get a stable URL.